components/Icons.tsx

```diff
@@ -0,0 +1,176 @@
+import { FC } from 'react';
+
+export type IconType = FC<{ fill?: string }>;
```

Comment:
How would IconType be used? It looks like the icons can have a default color, if I'm reading that right. Is this just a way to add more icon properties later?

Reply:
Its a way to add a map to the sidebar, just used to make typescript happy whenever inserting that stuff to the map and then iterating through it